app/views/home/index.html.erb

```diff
@@ -48,7 +48,9 @@
       </div>
       <div class="govuk-grid-row">
         <%= render partial: "layouts/collection_resources" %>
-        <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m govuk-!-margin-top-8 govuk-!-margin-bottom-8">
+        <div class="app-tab__underline">
+          <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m govuk-!-margin-top-8 govuk-!-margin-bottom-8">
```

Comment:
Can this class be a part of hr instead of wrapping it in a div?

Reply:
ah yep nice spot, thanks